packages/adminrouter/extra/src/test-harness/tests/test_master.py

```diff
@@ -351,3 +354,13 @@ def test_if_upstream_headers_are_correct(self,
                                              superuser_user_header,
                                              path_fmt.format(endpoint_type),
                                              )
+
+
+class TestNginxWorkersGroup:
```

Comment:
Can we please create a new file `test_common.py` for tests that are common for both master and agent ARs? This will match the way we want to dedup AR sources in general.

For now there is going to be only one test there, but as soon as we start writing tests for all the endpoints this number will grow.